src/js/modules/accounts/components/forms/SignOutForm.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import { reduxForm } from "redux-form";
+import injectSheet from "react-jss";
+
+import { signOut } from "../../actions";
+
+const styles = {
+  successMessage: {
+    color: "green",
+  },
+  errorMessage: {
+    color: "red",
+  },
+};
+
+const SignOutForm = ({ classes, handleSubmit, submitting, status }) => {
```

Comment:
Just curious, why is this a form?

Reply:
It's not just a button—also error and success message displaying.

Reply:
Okay, cool